.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -76,9 +76,9 @@ body:
   - type: dropdown
     id: pr
     attributes:
-      label: Are you willing to provide a PR?
+      label: (Optional) Will you be opening a PR for this?
```

Comment:
I understand the opening a PR is optional (ofc!) , but it's strange to mention `(Optional)` here since answering this question is required.
Maybe do not make this item required (change the value at line 86 below) and consider the default answer is false.

Reply:
Works for me.